doc_user/UserManual.adoc

```diff
@@ -2102,6 +2102,14 @@ image:image118c.png[Therapy,width=600,height=302]
 
 TIP: By pressing the *Worksheet* button, a visit can also be added there, after checking the actual workload for a specific date.
 
+[#visit-validation-rules]
+===== 8.10.8.1 Visits rules
```

Comment:
It seems we are using `visit` and `appointment` interchangeably, is there a preferred term?

Reply:
I think Appointment is the most suitable for the case

Reply:
Alright, I think it is the better term too.

Follow up question, should we also refactor the code to rename visit to appointment, for example `VisitManager` to `AppointmentManager`? I don't think it is urgent at all but I feel it is important to use a consistent vocabulary (as per  concept of [ubiquitous language](https://www.martinfowler.com/bliki/UbiquitousLanguage.html) )

Reply:
There are 92 occurrences of  the word `visit` in the doc, I'm not sure if we want to change all of them in this PR or just use `appointment` on new text.

Reply:
I updated this PR to favor appointment over visit but only on the part I added/updated.

Reply:
Thank you @joel-costigliola, I agree with the new issue, also because we have to bettere analyse it: some times the word "visit" refers to OPD visit, so in this case we have not to change it, hence the "ubiquitous" case. 

Is this PR completed or you want to add something?

Reply:
Fair enough @mwithi.

PR is complete and ready for review, I have moved the jira issue to `review doc` status